tools/fileinfo/detection/packers/vmprotect/test.py

```diff
@@ -0,0 +1,11 @@
+from regression_tests import *
+
+class Test(Test):
+    settings = TestSettings(
+        tool='fileinfo',
+        input='VMProtect-3.4_demo.ex'
+    )
+
+    def test_correctly_analyzes_input_file(self):
+        assert self.fileinfo.succeeded
+        assert self.fileinfo.output.contains(r'.*VMProtect \(2\.04\+\)*.*')
```

Comment:
Consider modifying the test in this way:
```diff
diff --git a/tools/fileinfo/detection/packers/vmprotect/test.py b/tools/fileinfo/detection/packers/vmprotect/test.py
index fb2e382..d360d01 100644
--- a/tools/fileinfo/detection/packers/vmprotect/test.py
+++ b/tools/fileinfo/detection/packers/vmprotect/test.py
@@ -3,9 +3,11 @@ from regression_tests import *
 class Test(Test):
     settings = TestSettings(
         tool='fileinfo',
-        input='VMProtect-3.4_demo.ex'
+        input='VMProtect-3.4_demo.ex',
+        args='--json'
     )

     def test_correctly_analyzes_input_file(self):
-        assert self.fileinfo.succeeded
-        assert self.fileinfo.output.contains(r'.*VMProtect \(2\.04\+\)*.*')
+        self.assertTrue(self.fileinfo.succeeded)
+        self.assertEqual(self.fileinfo.output['tools'][0]['name'], 'VMProtect')
+        self.assertEqual(self.fileinfo.output['tools'][0]['version'], '2.04+')
```
Reasons:
* The test will be more accurate (we look precisely at the name and version of the detected tool, not just grepping a string in the output).
* The test will more closely reflect what we actually want to test.
* No need for regexes :-) Yay!

Reply:
Sure, not a problem. I have used the asserts + regex to stay consistent to other tests I have checked.